ext/phar/config.w32

```diff
@@ -13,14 +13,9 @@ if (PHP_PHAR != "no") {
 		ADD_FLAG("CFLAGS_PHAR", "/D COMPILE_DL_PHAR ");
 	}
 	if (PHP_PHAR_NATIVE_SSL != "no") {
```

Comment:
> I think that can be simplified to `if (PHP_PHAR_SHARED) {` without loosing any functionality/flexibility.

Nope, might be better to leave that alone.

There is an issue with snapshot builds (which we are using to build the official Windows binaries), since ext/phar is built statically. As the PR is now, the core would have a dependency on OpenSLL, what we do not want.

Maybe the following is sufficient:

```suggestion
	if (PHP_PHAR_NATIVE_SSL != "no" && PHP_SNAPSHOT_BUILD == "no") {
```

This likely also solves the test failures. However, this may still cause issues, if ext/phar is built statically with native SSL support, and somebody loads php_openssl.dll (maybe built via phpize). And that might be a problem on non Windows systems as well.